### PR TITLE
Updated sensu-go vendored version to beta-8 and changed Entity.ID to Entity.Name

### PR DESCRIPTION
**Pull request by nikkictl**
Closes https://github.com/sensu/sensu-slack-handler/issues/1

Signed-off-by: Nikki Attea <nikki@sensu.io>

### CODE REVIEW

**Review by grepory — APPROVED** [review]

**Comment by grepory** [comment]
Let's give that a shot!

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+## [0.0.3] - 2018-11-21
+
+### Changed
```
Comment (echlebek) [inline CHANGELOG.md]: Technically, this is a breaking change for the handler.

If the project is to respect semver, the new version should probably be `0.1.0` or `1.0.0`.

Comment (nikkictl) [reply CHANGELOG.md]: Fair enough 👍

**Review by echlebek — APPROVED** [review]
:+1: :shipit:

**Comment by schampseix** [comment]
Many thanks for your jobs !

yakhatape

**Pull request merged**